backend/vaa-strapi/src/functions/utils/appSettings.ts

```diff
@@ -0,0 +1,73 @@
+import {type DynamicSettings, dynamicSettings} from 'vaa-shared';
+import {API} from './api';
+
+/**
+ * Gets `results.cardContents` from `dynamicSettings.ts` and returns them in format used in Strapi
+ */
+export function getCardContentsFromFile() {
```

Comment:
The `cardContents` settings are a bit involved, so some conversion is needed between the frontend type and the Strapi components.